skill.json

```diff
@@ -3,26 +3,26 @@
     "publishingInformation": {
       "locales": {
         "en-US": {
-          "summary": "Sample Short Description - powered by Rocket.Chat",
+          "summary": "Alexa Skill For Rocket.Chat",
           "examplePhrases": [
-            "Alexa open rocket chat",
-            "Alexa tell rocket chat hello",
-            "Alexa ask rocket chat to say hello"
+            "Alexa, Open Rocketchat",
+            "Alexa, Begin Rocketchat",
+            "Alexa, Begin Rocketchat"
           ],
-          "name": "Rocket Chat",
-          "description": "Sample Full Description - Innovating incredible new user experiences in the Alexa ecosystem - powered by Rocket.Chat"
+          "name": "Rocket.Chat",
+          "description": "Sample Full Description"
         }
       },
       "isAvailableWorldwide": true,
       "testingInstructions": "Sample Testing Instructions.",
-      "category": "COMMUNICATION",
+      "category": "KNOWLEDGE_AND_TRIVIA",
```

Comment:
COMMUNICATION   or  COLLABORATION  or MESSAGINE  or CHAT 

will be more appropriate

Reply:
Oh must have slipped from base starter document no problem changed it